frontend/src/components/about/Value.js

```diff
@@ -1,78 +1,65 @@
-                width: 1 / 2,
-                display: 'flex',
-                flexDirection: 'column',
-                px: 4,
-                mb: 1
-            }} >
-
-                <Typography mb={1} color='primary.main' fontSize={20}> {valueObj.values[i - 1].title}</Typography>
-                <Typography color='secondary.main' fontSize={14}> {valueObj.values[i - 1].body}</Typography>
-            </Box>
-        </Box>))
+for (var i = 1; i < (valueObj.values.length + 1); i++) {
+    if (i%2 == 0) {
+        // right value
+        displayDirectionVal = 'row'
+        textAlignVal = 'right'
+        marginVal = -20
 
+    } else {
+        // left value
+        displayDirectionVal = 'row-reverse'
+        textAlignVal = 'left'
+        marginVal = 20
     }
-    //Right value
-    else if (i % 2 === 0) {
-        display.push((<Box sx={{
-            display: 'flex',
-            flexDirection: 'row',
-            width: 1,
-            height: 1,
-            alignItems: 'center',
-            justifyContent: 'center',
-            px: 5,
-            bgcolor: 'primary.light',
-            color: 'primary.dark'
-
-        }} key={i}>
-
 
+    display.push((
+    <Grid container direction={displayDirectionVal} textAlign={textAlignVal} justifyContent='flex-end' ml={marginVal} sx={{
+        bgcolor: 'primary.light',
+        color: 'primary.dark'
+    }} key={i}>
+        <Grid item xs={6}>
             <Box sx={{
-                width: 1 / 2,
-                display: 'flex',
-                flexDirection: 'column',
-                px: 4,
-                mb: 1
+                mx: 10
             }}>
+                <Typography gutterBottom={true} color='primary.main' variant='h5' sx={{
+                    fontWeight: 'bold'
+                }}> {valueObj.values[i - 1].title}</Typography>
+
+                <Typography color='secondary.main' variant='body1' sx={{
+                }}> {valueObj.values[i - 1].body}</Typography>
 
-                <Typography mb={1} align='right' color='primary.main' fontSize={20}> {valueObj.values[i - 1].title}</Typography>
-                <Typography align='right' color='secondary.main' fontSize={14}> {valueObj.values[i - 1].body}</Typography>
             </Box>
-            <img src={valueObj.values[i - 1].image} width={120} height={120} />
-        </Box>))
+        </Grid>
+
+        <Grid item xs={1}>
+            <Box component='img' src={valueObj.values[i - 1].image} alt="Unable to load image" sx={{
+                width: 140,
+                height: 140,
+            }}/>
+        </Grid>
+
+    </Grid>))
 
     }
-}
 
 function Value() {
     return (
         <main>
-            <Typography color="primary.dark" ml={30} fontSize={25} fontWeight="bold"> Our Values </Typography>
+            <Typography color="primary.dark" variant='h4' sx={{
+                p: 2,
+                pl: 4,
+                fontWeight: 'bold'
+            }}> Our Values </Typography>
```

Comment:
we should add more bottom margin from the "Our Values" subtitle to the list of values